### PR TITLE
[documentation] README.md to use add package instead of install

### PR DESCRIPTION
**Pull request by chamons**
- dotnet install does not work on my machine, but dotnet add package does
- We have 51 references to 'add package' and 9 for 'install' in readme right now

### CODE REVIEW

File: sdk/keyvault/Azure.Security.KeyVault.Certificates/README.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ The Azure Key Vault certificates client library enables programmatically managin
 Install the Azure Key Vault certificates client library for .NET with [NuGet][nuget]:
 
 ```PowerShell
-dotnet install Azure.Security.KeyVault.Certificates -IncludePrerelease
+dotnet add package Azure.Security.KeyVault.Certificates -IncludePrerelease
```
Comment (pakrym) [inline sdk/keyvault/Azure.Security.KeyVault.Certificates/README.md:14]: I don't think the `IncludePrerelease` is a valid `dotnet add package` flag

Comment (chamons) [reply sdk/keyvault/Azure.Security.KeyVault.Certificates/README.md:14]: Let me check, I blindly converted here.

Comment (chamons) [reply sdk/keyvault/Azure.Security.KeyVault.Certificates/README.md:14]: It should be `--prerelease`. I will correct. Apologies.

Comment (chamons) [reply sdk/keyvault/Azure.Security.KeyVault.Certificates/README.md:14]: I will also test each one.

File: sdk/keyvault/Azure.ResourceManager.KeyVault/README.md [hunk 2]
```diff
@@ -9,7 +9,7 @@ This package follows the [new Azure SDK guidelines](https://azure.github.io/azur
 Install the Azure Key Vault management library for .NET with [NuGet](https://www.nuget.org/):
 
 ```PowerShell
-dotnet install Azure.ResourceManager.KeyVault -Version 1.0.0-preview.2 
+dotnet add package Azure.ResourceManager.KeyVault -Version 1.0.0-preview.2
```
Comment (pakrym) [inline sdk/keyvault/Azure.ResourceManager.KeyVault/README.md:12]:
```suggestion
dotnet add package Azure.ResourceManager.KeyVault -version 1.0.0-preview.2
```

File: sdk/keyvault/Azure.Security.KeyVault.Certificates/README.md (hunk 1 above)
Comment (pakrym) [reply sdk/keyvault/Azure.Security.KeyVault.Certificates/README.md:14]: `prerelease` is .NET 5 specific flag. Let's use the `--version` parameter for now.

```suggestion
dotnet add package Azure.Security.KeyVault.Certificates --version 4.2.0-beta.3
```

**Review by pakrym — APPROVED** [review]

**Comment by chamons** [comment]
Hey @pakrym I tried testing your suggestion and I get:
```
 dotnet add package Azure.ResourceManager.KeyVault --version 1.0.0-preview.2
  Determining projects to restore...
  Writing /var/folders/gc/7vct49652r1df_tp36ffvph00000gn/T/tmpGUnEYS.tmp
info : Adding PackageReference for package 'Azure.ResourceManager.KeyVault' into project '/Users/donblas/tmp/nuget/nuget.csproj'.
info : Restoring packages for /Users/donblas/tmp/nuget/nuget.csproj...
info :   CACHE https://api.nuget.org/v3-flatcontainer/azure.resourcemanager.keyvault/index.json
error: NU1102: Unable to find package Azure.ResourceManager.KeyVault with version (>= 1.0.0-preview.2)
error:   - Found 1 version(s) in nuget.org [ Nearest version: 1.0.0-preview.1 ]
error:   - Found 0 version(s) in nuget Local
error: Package 'Azure.ResourceManager.KeyVault' is incompatible with 'all' frameworks in project '/Users/donblas/tmp/nuget/nuget.csproj'.
```

Should I swap back to --prerelease?

**Comment by chamons** [comment]
All but ^ are confirmed to work on a new project.

**Comment by pakrym** [comment]
>Should I swap back to --prerelease?

Nah, the version in the readme is wrong. We never shipped a preview.2, let's instead use the `dotnet add package Azure.ResourceManager.KeyVault --version 1.0.0-preview.1`

File: sdk/keyvault/Azure.ResourceManager.KeyVault/README.md (hunk 2 above)
Comment (heaths) [reply sdk/keyvault/Azure.ResourceManager.KeyVault/README.md]: Actually, this has to be `--version` (two dashes).

**Pull request merged**